### ISSUE TITLE
Unable to upload image from Chrome

### ISSUE THREAD

**Issue by xiasu** [makeabilitylab/makeabilitylabwebsite#1011]
When I try to upload image using our backend webpage, the "Choose File" button is not working. This only happens recently, on Chrome only. I switched to Safari and there the button does work.

https://user-images.githubusercontent.com/61726267/195467318-6342e7ee-0610-4e2b-a40d-6e90262a8957.mov

**Comment by jonfroehlich** [comment]
Can you say which page you were on? And what the "Choose File" button is for?

As much information as you can supply would be greatly appreciated.

I just checked and it works for me but I don't know if I'm on the right page because your description is unclear.

![image](https://user-images.githubusercontent.com/1621749/195481474-75e6e96e-7893-4d41-84a5-c763b9aab68e.png)

**Comment by xiasu** [comment]
I tried again on my mac, **every** "choose file" button I can find is not working.
For example, this one on the Banner edit page.
![image](https://user-images.githubusercontent.com/61726267/195504497-69ac4701-78b8-4115-baf6-e5d664c1f673.png)
It might be due to some local browser compatibility issue. But it's still very weird since the button used to work, but suddenly failed since Tuesday morning, when I was editing Kaiming's profile. I first added his information without an image, then tried to edit the profile to add one, that's when the "Choose File" button started to fail, and it never worked since then.
![image](https://user-images.githubusercontent.com/61726267/195505295-5fb583fe-d84a-4281-87c2-2d5ddd3e6116.png)
